wandb/sdk/wandb_settings.py

```diff
@@ -917,11 +920,16 @@ def _run_url(self) -> str:
         query = self._get_url_query_string()
         return f"{project_url}/runs/{quote(self.run_id)}{query}"
 
-    def _start_run(self) -> None:
+    def _start_run(self, source: int = Source.BASE) -> None:
```

Comment:
nit: name seems weird given that we only time stamp this

Reply:
Agreed. I just kept the way it was called previously. How about `_set_start_times`?

Reply:
yeah makes more sense...

Reply:
I'd keep a private method, but would be happy to move the call in `wandb.init()` to `_apply_init`. @raubitsj there's a todo in that place from you saying "# TODO(jhr): should this be moved? probably.", so I'd defer it to you to make the final decision thanks :)
Also, it's used in `standalone_tests/grpc_client.py`.